{{cookiecutter.project_slug}}/requirements/test.txt

```diff
@@ -2,9 +2,11 @@
 
```

Comment:
This militates for Docker-for-dev, perhaps. It's one thing for us to do that among Oddbird (and I don't mean a simple thing, just… a thing), but would that be good for this template?

(I say it militates for it because Docker seems like a viable way to isolate your Node and Python versions for a development task really well, requiring nothing more than a version number specification in a Dockerfile, and a bit of time to download it initially.)

Reply:
@davisagli That's reasonable. Ideally I'd like to have the versions of node/yarn customizable too, when setting up the cookiecutter project. Just wasn't sure if it was a priority (there's a long list of things that _could_ be cookiecutter settings)...